__tests__/discussion/seeds/getUsers.js

```diff
@@ -0,0 +1,10 @@
+const zeroFill = require('zero-fill')
```

Comment:
You could use padStart to do zero filling (implemented since node 8):

http://node.green/#ES2017-features-String-padding-String-prototype-padStart